pkg/component/monitoring/charts/seed-monitoring/charts/core/charts/prometheus/rules/worker/kube-pods.rules.yaml

```
@@ -35,7 +35,7 @@ groups:
       description: Pod {{ $labels.pod }} is not ready for more than 1 hour.
       summary: Shoot pod is in a not ready state
   - alert: KubePodNotReadyControlPlane
-    expr: kube_pod_status_ready{condition="true", type="seed", pod!~"(.+)curator(.+)"} == 0
+    expr: kube_pod_status_ready{condition="true", type="seed", pod!~"(.+)(curator|compact-job)(.+)"} == 0
```

I think this `curator` thingy is from the times we had a curator CronJob to clean up elasticsearch indices. See https://github.com/ialidzhikov/gardener/blob/v0.35.0/charts/seed-bootstrap/charts/elastic-kibana-curator/templates/curator-es/curator-cronjob.yml.

I can be removed now long time ago we replaced elasticsearch with loki.
```suggestion
    expr: kube_pod_status_ready{condition="true", type="seed", pod!~"(.+)compact-job(.+)"} == 0
```

---

Thanks, removed both occurences.